### ISSUE TITLE
[14] Components are still keyboard accessible when not visible

### ISSUE THREAD

**Issue by gordcorp** [AusDTO/dta-website#287]

**Comment by bjgordon** [comment]
@klepas to fix in uikit 2.x

**Comment by bjgordon** [comment]
Reported to uikit https://github.com/AusDTO/gov-au-ui-kit/issues/476
